sadu-queries/src/main/java/de/chojo/sadu/wrapper/mapper/RowMappers.java

```diff
@@ -0,0 +1,162 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 RainbowDashLabs and Contributor
+ */
+
+package de.chojo.sadu.wrapper.mapper;
+
+import de.chojo.sadu.wrapper.mapper.exceptions.MappingAlreadyRegisteredException;
+import de.chojo.sadu.wrapper.mapper.exceptions.MappingException;
+import de.chojo.sadu.wrapper.mapper.rowmapper.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Class to register {@link RowMapper} to map rows to objects.
+ */
+public class RowMappers {
```

Comment:
I think "MapperRegistry" similar would be less confusing. RowMappers sounds like a name of a field of type list.

Reply:
Renamed it to RowMappersRegistry